src/Routes.php

```diff
@@ -43,4 +43,19 @@ public static function customers()
 
         return $anonymous;
     }
+
+    public static function bankAccounts()
```

Comment:
Missing docblock

Reply:
No one of these functions have docblock, do you want me to implement that?

Reply:
I see. And I've did that at #285, so, if you could only add the docblock to `bankAccounts` it woill help us a lot :smile: